its/ruling/src/integrationTest/resources/sources/kubernetes_cross_file/pod.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-ns-compliant
+  namespace: default-mem-example
+spec:
+  containers:
+    - name: nginx-ns-compliant
+      image: nginx
+    # Expect S6869 and S6864
```

Comment:
It looks like this comment is not valid as files with expected issues are not modified.